src/others/contexts/file-downloader.tsx

```diff
@@ -97,8 +97,16 @@ export const FileDownloaderContextProvider: FunctionComponent = ({ children }) =
 
       const csvRow = [requestDate];
 
+      const categoryToAmount: { [id: string]: number } = {};
       cityRequests.forEach((request) => {
-        csvRow.push(request.amount + "");
+        categoryToAmount[request.category] = request.amount;
+      });
+      uniqueCategories.forEach((category) => {
+        if (category in categoryToAmount) {
+          csvRow.push(categoryToAmount[category] + "");
+        } else {
+          csvRow.push("0");
```

Comment:
**question**

Why "0"?

Reply:
What instead? Empty?

Reply:
Just `csvRow.push(categoryToAmount[category]`. It should be 0 if it's 0. So that if - else seems redundant know when I am thinking of it.

Also: `categoryToAmount[category] + ""` => String(`categoryToAmount[category]) reads better, isn't it?

Reply:
So you don't think that a "0" value as a default makes sense for that csv?

Like if noone requested "water" on that date in that city, then we have 0 in that cell instead of empty. No?

If you don't think that is useful then for sure i can change the 5 lines into a single one like you suggested.

Reply:
I totally think that the default for the amount should be `0`. 
I only question setting it here and ask why it's not set to 0 where the total is calculated.